karibu-dsl/src/main/kotlin/com/github/mvysny/karibudsl/v10/KLitRendererDsl.kt

```diff
@@ -0,0 +1,166 @@
+package com.github.mvysny.karibudsl.v10
+
+import com.github.mvysny.karibudsl.v10.KLitRendererBuilderA.Property
+import com.vaadin.flow.data.renderer.LitRenderer
+import kotlin.properties.ReadOnlyProperty
+import kotlin.reflect.KProperty
+
+
+/**
+ * PRB: \${item.itemName} -> isn't interpreted by [LitRenderer]
```

Comment:
I'm not sure what this means - could you please rephrase so that users know how to use this function, and what it does?